### PR TITLE
Nikita nair

### PR DESCRIPTION
**Pull request by NikitaN18**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,13 +4,54 @@ The CHAMMI dataset is available for download on [Zenado](https://zenodo.org/reco
 
 We released model checkpoints and the training scripts to further aid reproducibility. 
 Our code and checkpoints are now publicly available at [here](https://github.com/chaudatascience/channel_adaptive_models).
+
+## Disclosure
```
Comment (jccaicedo) [inline README.md:8]: Remove the title

File: README.md [hunk 2]
```diff
@@ -4,13 +4,54 @@ The CHAMMI dataset is available for download on [Zenado](https://zenodo.org/reco
...
+
+These instructions are made for conda environments, but the `requirements.txt` file is given for other users
+
+### Notice
```
Comment (jccaicedo) [inline README.md:12]: Move this title up to include the previous statement.
Change the title from Notice to Note (keep the three hashes)

File: README.md [hunk 3]
```diff
@@ -4,13 +4,54 @@ The CHAMMI dataset is available for download on [Zenado](https://zenodo.org/reco
...
+
+Check python version by running
+```
+python --version
+```
+
+If your python version is not 3.8 or 3.9, install python with
+`conda install python=3.8` or `conda install python=3.9`
+
+### Recommended Version:
```
Comment (jccaicedo) [inline README.md:28]: Remove it

File: README.md [hunk 4]
```diff
@@ -4,13 +4,54 @@ The CHAMMI dataset is available for download on [Zenado](https://zenodo.org/reco
...
+
+
+## Checking and installing correct python version
+
+Check python version by running
+```
+python --version
+```
+
+If your python version is not 3.8 or 3.9, install python with
```
Comment (jccaicedo) [inline README.md:25]: Update to recommend 3.9.

File: README.md [hunk 5]
```diff
@@ -4,13 +4,54 @@ The CHAMMI dataset is available for download on [Zenado](https://zenodo.org/reco
...
+`conda install python=3.8` or `conda install python=3.9`
+
+### Recommended Version:
+
+```
+conda install python=3.9
+```
+
+#### Notice
```
Comment (jccaicedo) [inline README.md:34]: Remove the little titles and only keep the major sections

**Review by jccaicedo — CHANGES_REQUESTED** [review]
This is looking great! Please address the minor changes in the README file.

**Comment by NikitaN18** [comment]
Thanks! I made the necessary changes to the README file and made it more organized.

**Review by jccaicedo — APPROVED** [review]
Looks good to me!

**Pull request merged**